src/main/java/org/openlmis/fulfillment/web/util/StockEventBuilder.java

```diff
@@ -255,4 +255,8 @@ private UUID getNodeId(UUID fromFacilityId, UUID toFacilityId, UUID programId,
     throw new ValidationException(EVENT_MISSING_SOURCE_DESTINATION, toFacility.getCode());
   }
 
+  private String getUserSignature() {
+    return authenticationHelper.getCurrentUser().getFirstName() + " " +
```

Comment:
use StringBuilder

Reply:
name method differently, you are building/creating user signature, not getting

Reply:
do not retrieve current user 3 times in the method, performance problem

Reply:
instead of " " try to use constant from util classes

Reply:
what if firstNAme or lastNAme is null ?

Reply:
> use StringBuilder

Fixed

Reply:
All reported comments has been fixed